scripts/collect_images.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import datetime
+import json
+import os
+import subprocess
+
+from src.db.db_connection import sql_cursor
+
+"""
+This script should be set up with a cron job to run daily.
+rclone should be set up, in our case pointing to a Google Drive folder: https://rclone.org/drive/
+
+Collect images of non-goggle detections from the database.
+Upload images and metadata to Google Drive.
+"""
+
+METADATA_FILE = 'metadata.json'
+# rclone on ee220clnx1 is an earlier version that doesn't support copying to shared folders
+RCLONE_PATH = '/home/shay/a/bergz/rclone-v1.52.2-linux-amd64/rclone'
+TODAY = datetime.datetime.today().strftime('%Y-%m-%d')
```

Comment:
dates usually need to be calculated in the function they are called in. However I understand that this is a simple script which probably doesn't run for a very long period of time. If that is the case, please leave a comment denoting that so this doesn't become a future hidden bug.